datasets/gb/coh/gb_coh_disqualified.yml

```diff
@@ -69,8 +69,16 @@ lookups:
           - Other
           - Not Known
           - Unknown
+          - Not Disclosed
         value: null
       - match: Botswanan
         value: Botswana
       - match: Citizen Of Seychelles
         value: Seychelles
+      - match: 
+          - Brithish
+        value: Great Britain
+  type.date:
+    options:
+      - match: "3018-12-12"
+        value: "2018-12-12"
```

Comment:
any idea what this was for? is it safe to assume this is what they meant?

Reply:
some entites has ["2018-12-12"] either as a startDate or a listingDate, I'd assume that's the case